src/app/features/genes/components/gene-hero/gene-hero.component.spec.ts

```diff
@@ -110,4 +110,14 @@ describe('Component: Gene Hero', () => {
     const expected = 'Immune Response, Lipid Metabolism, Structural Stabilization, Synapse, Vasculature';
     expect(component.getBiodomains()).toBe(expected);
   });
+
+  it('should return the ensembl permalink', () => {
+    component.gene = geneMock1;
+    expect(component.getEnsemblUrl()).toBe('https://may2015.archive.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000264794');
+  });
```

Comment:
What do you think about adding a test case for a gene without an `ensembl_permalink`?

Reply:
good idea, i'll add that.